src/components/Points/CampaignPromo.tsx

```diff
@@ -102,7 +102,7 @@ export const CampaignPromo = () => {
         </Typography>
         <Typography variant="body2">& Safe points</Typography>
         <Typography variant="body2" color="text.secondary" mt={5}>
-          Learn more and participate in campaign:
+          <ExternalLink href="https://dub.sh/morphocampaign">Learn more</ExternalLink> and participate in campaign:
```

Comment:
Do we have control over this shortened url?

Reply:
Yes the governance team should have control over this.